src/lib/elements/ErrorMessage.js

```diff
@@ -0,0 +1,92 @@
+// This file is part of React-Invenio-Forms
+// Copyright (C) 2023 CERN.
+//
+// React-Invenio-Forms is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import { Message, Icon } from "semantic-ui-react";
+import PropTypes from "prop-types";
+import React, { Component } from "react";
+import _isEmpty from "lodash/isEmpty";
+
+const FieldErrorList = ({ fieldErrors }) => {
+  return (
+    <Message.List>
+      {fieldErrors.map((error) => {
+        return (
+          <Message.Item key={error}>
+            {/* when there is no field Marshmallow returns _schema */}
+            {error.field !== "_schema" && <strong>{error.field}: </strong>}
+            {error.messages.length === 1 ? (
+              error.messages[0]
+            ) : (
+              <Message.List>
+                {error.messages.map((message) => (
+                  <Message.Item key={message}>{message}</Message.Item>
+                ))}
+              </Message.List>
+            )}
+          </Message.Item>
+        );
+      })}
+    </Message.List>
+  );
+};
+
+FieldErrorList.propTypes = {
+  fieldErrors: PropTypes.array.isRequired,
+};
+
+/**
+ * General component for error messages.
+ * Handles:
+ *   - simple error messages: provide custom header, and content - response message of received error;
+ *   - complex error messages: provide list or received errors.
+ * Icon and other ui props are supported.
+ */
+export class ErrorMessage extends Component {
+  render() {
+    const { header, errors, content, icon, ...uiProps } = this.props;
+
+    return (
+      <Message icon {...uiProps}>
+        {icon && <Icon name={icon} />}
+        <Message.Content>
```

Comment:
```suggestion
        <Message.Content role="alert">
```

Actually, one comment! I think this should have `role="alert"`, but it needs to be tested with the screenreader.